documentation/modules/adding-source-provider.adoc

```diff
@@ -21,35 +21,126 @@ You can add {a-rhv} source provider by using the {ocp} web console.
+
+.Prerequisites
+
+* TBD
+endif::[]
 
 .Procedure
 
 . In the {ocp} web console, click *Migration* -> *Providers for virtualization*.
 . Click *Create Provider*.
 ifdef::vmware[]
-. Select *VMware* from the *Provider type* list.
+. Click *VMware vSphere*.
 . Specify the following fields:
 
-* *Provider name*: Name to display in the list of providers
-* *vCenter host name or IP address*: vCenter host name or IP address - if a certificate for FQDN is specified, the value of this field needs to match the FQDN in the certificate
-* *vCenter user name*: vCenter user, for example, `user@vsphere.local`
-* *vCenter password*: vCenter user password
-* *VDDK init image*: VDDKInitImage path
+* *Provider resource name*: Name to display in the list of providers.
+* *URL*: Endpoint.
+* *VDDK init image*: VDDKInitImage path.If left empty, some functionality will not be available.
+* *Username*: vCenter user, for example, `user@vsphere.local`.
+* *Password*: vCenter user password.
+* *SHA-1 fingerprint*: [TBD.
 
 . To allow a migration without validating the provider's CA certificate, select the *Skip certificate validation* check box. By default, the checkbox is cleared, meaning that the certificate will be validated.
-. Enter the *SHA-1 fingerprint*.
 endif::[]
 ifdef::rhv[]
-. Select *Red Hat Virtualization* from the *Provider type* list.
+. Click *Red Hat Virtualization*.
 . Specify the following fields:
 
-* *Provider name*: Name to display in the list of providers
-* *RHV Manager host name or IP address*: {manager} host name or IP address -  if a certificate for FQDN is specified, the value of this field needs to match the FQDN in the certificate
-* *RHV Manager user name*: {manager} user
-* *RHV Manager password*: {manager} password
+* *Provider resource name*: Name to display in the list of providers
+* *URL*: Endpoint
+* *Username*: Username
+* *Password*: Password
 
 . To allow a migration without validating the provider's CA certificate, select the *Skip certificate validation* check box. By default, the checkbox is cleared, meaning that the certificate will be validated.
-. If you did not select *skip certificate validation*, the *CA certificate* field is visible. Drag the CA certificate to the text box or browse for it and click *Select*. Use the {manager} CA certificate or {manager} Apache CA certificate if the {manager} CA certificate was replaced by a third-party certificate on the Apache server. If you did select the check box, the *CA certificate* text box is not visible.
+. If you did not select *skip certificate validation*, the *CA certificate - leave empty to use system certificates* field is visible. Drag the CA certificate to the text box or browse for it and click *Select*. Use the {manager} CA certificate or {manager} Apache CA certificate if the {manager} CA certificate was replaced by a third-party certificate on the Apache server. If you did select the check box, the *CA certificate* text box is not visible.
```

Comment:
we need better phrasing here to be clear about the user options
- skip verification
- don't skip verification and use system certs
- don't skip verification and use custom certs

Reply:
@yaacov @ahadas As far as i can tell, we _validate_ certificates and _verify_ signatures.  @anarnold97 Do you agree?

Reply:
@yaacov @ahadas 

How's this? [also in the file and in the preview now]

. Choose one of the following options for validating a CA certificate:

** *Skip validation* : Migrate without validating a CA certificate.
** *Use the system CA certiicate*: Migrate after validating the system CA certificate.
** *Use a custom CA certificate*: Migrate after validating a custom CA certificate.

.. To skip validation, select the *Skip certificate validation* check box.
.. To validate the system CA certfiticate, leave the *Skip certificate validation* check box cleared and leave the *CA certificate* text box empty.
.. To validate a custom CA certificate, leave the *Skip certificate validation* check box cleared and _either_ drag the CA certificate to the text box _or_ browse for it and click *Select*.

Reply:
@RichardHoch - i do like the additional formatting as it looks much cleaner and is easier to read.

@yaacov & @ahadas - this cool with you?

Reply:
@yaacov @ahadas  Is this wording OK?

Reply:
> @RichardHoch - i do like the additional formatting as it looks much cleaner and is easier to read.

+1

> @yaacov @ahadas Is this wording OK?

it looks good besides the `the system CA certificate` part as it suggests there's a single system certificate, I'd change it to `the system CA certificates`